native/chat/chat-thread-list.react.js

```diff
@@ -255,8 +259,17 @@ const styles = {
 };
 const stylesSelector = styleSelector(styles);
 
-export default connect((state: AppState) => ({
+export const HomeChatThreadList = connect((state: AppState) => ({
   chatListData: chatListData(state),
+  searchChatListData: chatBackgroundListData(state).concat(chatListData(state)),
+  viewerID: state.currentUserInfo && state.currentUserInfo.id,
+  threadSearchIndex: threadSearchIndex(state),
+  styles: stylesSelector(state),
+}))(ChatThreadList);
+
+export const BackgroundChatThreadList = connect((state: AppState) => ({
```

Comment:
Small nit - with the exception of files with tons of exports, such as type definition files, I prefer to do all the exports at once at the end of the file, eg. `export { HomeChatThreadList, BackgroundChatThreadList }`